src/components/Footer/Logo/Logo.test.tsx

```diff
@@ -29,28 +26,4 @@ describe('Logo component', () => {
     const { getByText } = render(<Logo image={logoImage} heading={heading} />)
     expect(getByText('Swoosh Branding')).toBeInTheDocument()
   })
```

Comment:
we should still keep some kind of test of the `size` prop influencing classes for Logo

Reply:
Oh nice catch, yea this test file doesn't have tests for just size by itself, so instead of deleting that code I'll repurpose it to be similar to what we have for other components

Reply:
[dffb9a5](https://github.com/trussworks/react-uswds/pull/1988/commits/dffb9a5c255796173d5c79e6d8e0ef88e1956357)